.devops/deploy-pipelines.yml

```diff
@@ -57,6 +57,12 @@ resources:
       name: pagopa/azure-pipeline-templates
       ref: refs/tags/v4
       endpoint: 'pagopa'
+    
+    - repository: templates
+      type: github
+      name: pagopa/azure-pipeline-templates
+      ref: refs/tags/v9
+      endpoint: 'pagopa'
```

Comment:
I strongly agree with the decision of having 2 references for the same template collections, so that we can add the feature of `v9` without being forced to upgrade other templates. I just think we need this motivation to be explicit, we owe this to our future selves. 
Suggestions:
* use a comment block explaining why we use 2 different versions of the same thing
* do not use `pagopaCommons` and `template`: they're totally different names and seems to be unrelated. We may want to call them, respectively:
  1. `pagopaCommonsV4` and `pagopaCommonsV9`, or else 
  1. `pagopaCommonsLegacy` and `pagopaCommons`
  1. whatever you find more appropriate
  (I vote for _2_ as it emphasises the fact that we need to get rid of the legacy reference)
  
 What do you think?

Reply:
The second is fine with me too, so we are not bound by the version